prov/shm/src/smr.h

```diff
@@ -161,6 +161,16 @@ struct smr_domain {
 	int			ep_idx;
 };
 
+#define SMR_PREFIX	"fi_shm://"
+#define SMR_PREFIX_NS	"fi_ns://"
```

Comment:
The commit message for this change is the most beautiful commit message I've ever read.  It nearly brought a tear to my eye.  Please insert the message as a section in the shm man page on addressing, or into some other documentation that will allow a user to understand how shm addressing works.  Then this will not be buried in the git history abyss.

Reply:
I thought you would enjoy that! I haven't updated the man page since the original commit. Now that it's pretty stable and functional, I'm working on updating the man pages with a more accurate description of the provider. I'll make sure to include this commit message in that update.